test/Spec.hs

```diff
@@ -1,17 +1,34 @@
 module Main where
 
 import Control.Exception (bracket)
+import Hedgehog (Group (..), checkParallel)
 import System.IO (hSetEncoding, utf8)
+import Test.Hspec (Spec, hspec)
+import Test.Hspec.Core.Spec (sequential)
 
 import IW (mkAppEnv)
 import IW.App (AppEnv, Env (..))
 import IW.Config (loadConfig)
 import IW.Db (prepareDb)
 import IW.Effects.Log (runAppLogIO_)
 
+import Test.Common (joinSpecs)
+import Test.Core.Issue (issueInsertAndQuery)
+
 import qualified Data.Pool as Pool
 
 
+hspecTests :: AppEnv -> Spec
+hspecTests = sequential . joinSpecs "issue-wanted" []
+
+hedgehogTests :: AppEnv -> Group
+hedgehogTests env = Group "Roundtrip properties" 
+    [ issueInsertAndQuery env `named` "issues === (insert . query $ issues)"
```

Comment:
The text here is outdated a little bit. It's not a single issue. Also, for simplicity reasons I propose to not mention `query` here. It's an implementation detail of how we test roundtrip properties. So the text can be just:

```haskell
fromRow . toRow ≡ id
```